src/frontend/planner_test/tests/testdata/output/subquery.yaml

```diff
@@ -177,15 +177,20 @@
     └─LogicalProject { exprs: [1:Int32] }
       └─LogicalJoin { type: Inner, on: true, output: all }
         ├─LogicalAgg { aggs: [] }
```

Comment:
It looks like we can optimize `Join` with `Agg { aggs: [] }` as one side to its other side.